ichnaea/alembic/versions/000000000000_base.py

```diff
@@ -29,60 +28,13 @@
     BASE_SQL = fd.read().strip()
 
 
-def _db_creds(conn_uri):
-    # for example 'mysql+pymysql://user:pwd@localhost/location'
-    result = conn_uri.split("@")[0].split("//")[-1].split(":")
-    return DBCreds(*result)
-
-
-def _add_users(conn):
-    # We don't take into account hostname or database restrictions
-    # the users / grants, but use global privileges.
-    creds = {}
-    creds["rw"] = _db_creds(settings("db_readwrite_uri"))
-    creds["ro"] = _db_creds(settings("db_readonly_uri"))
-
-    stmt = text("SELECT user FROM mysql.user")
-    result = conn.execute(stmt)
-    userids = set([r[0] for r in result.fetchall()])
-
-    create_stmt = text("CREATE USER :user IDENTIFIED BY :pwd")
-    grant_stmt = text("GRANT delete, insert, select, update ON *.* TO :user")
-    added = False
-    for cred in creds.values():
-        if cred.user not in userids:
-            conn.execute(create_stmt.bindparams(user=cred.user, pwd=cred.pwd))
-            conn.execute(grant_stmt.bindparams(user=cred.user))
-            userids.add(cred.user)
-            added = True
-    if added:
-        conn.execute("FLUSH PRIVILEGES")
```

Comment:
I decided it's not great to have the migrations look at the environment variables and then add mysql db users. Seems like it's better for an ops person to explicitly add them for server environments. For local dev and testing environments, we just use the root account.